src/javaMain/kotlin/io/github/oshai/kotlinlogging/slf4j/internal/Slf4jMarker.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.oshai.kotlinlogging.slf4j.internal
+
+import io.github.oshai.kotlinlogging.Marker
+
+@JvmInline
```

Comment:
I am not sure there is a real performance gain here from inline. it might even be the opposite because of boxing / unboxing. So unless there is an obvious reason I suggest to have it a regular class.